### DISCUSSION TITLE
Apollo supported records!

### DISCUSSION THREAD

**Discussion by daohu527** [ApolloAuto/apollo#15025] [General]
1. Use recorded record files

| Record name       | Verification                                              | Vehicle configuration      | Map            | Size   | Download address                                             |
| ----------------- | --------------------------------------------------------- | -------------------------- | -------------- | ------ | ------------------------------------------------------------ |
| demo_3.5.record   | record replay                                             | MKZ | Sunnyvale Loop        | 91.8 MB | https://apollo-system.cdn.bcebos.com/dataset/6.0_edu/demo_3.5.record |
| sensor_rgb.tar.xz | camera_detection、lidar_detection、trafficlight_detection | Perception Test V1         | Sunnyvale Loop | 4.4GB  | https://apollo-system.bj.bcebos.com/dataset/6.0_edu/sensor_rgb.tar.xz |

2. Convert from dataset(nuScenes, KITTI)

Through the [adataset](https://pypi.org/project/adataset/) tool, the dataset can be converted into Apollo record file. You can follow the readme to convert records.

we will provide a form similar to the one above, because records are often related to vehicle configuration and maps.

Hope it helps you!

**Comment by freekevinCode** [comment]
mark

**Comment by boyang9602** [comment]
Hi @daohu527 ,

I would like to confirm with you about the vehicle configuration of the sensor_rgb record. The record data contains only velodyne64 point cloud. However, the perception_test_v1 configuration only has velodyne128 parameters. Should I just use the velodyne128 parameters for the LiDAR in sensor_rgb? I am working on the localization module btw. Thank you very much!

Best,
Bo Yang

---
Info of sensor_rgb.record (point cloud channel name indicates it is velodyne64)
```
bo@in_dev_docker:/apollo$ cyber_recorder info records/sensor_rgb.record 
WARNING: Logging before InitGoogleLogging() is written to STDERR
I0728 22:22:08.331987 58590 global_data.cc:150] [cyber_recorder] host ip: 192.168.170.9
record_file:    records/sensor_rgb.record
version:        1.0
duration:       60.901905 Seconds
begin_time:     2018-01-03 06:37:30
end_time:       2018-01-03 06:38:31
size:           7626958089 Bytes (7.103158 GB)
is_complete:    true
message_number: 43820
channel_number: 14
channel_info:                   /tf_static                                               1 messages: apollo.transform.TransformStampeds
                /apollo/sensor/gnss/imu                              11630 messages: apollo.drivers.gnss.Imu
                /tf                                                  11740 messages: apollo.transform.TransformStampeds
                /apollo/canbus/chassis                                5851 messages: apollo.canbus.Chassis
                /apollo/sensor/gnss/odometry                          5848 messages: apollo.localization.Gps
                /apollo/localization/pose                             5856 messages: apollo.localization.LocalizationEstimate
                /apollo/sensor/conti_radar                             789 messages: apollo.drivers.ContiRadar
                /apollo/sensor/gnss/rtk_obs                            352 messages: apollo.drivers.gnss.EpochObservation
                /apollo/sensor/velodyne64/compensator/PointCloud2      587 messages: apollo.drivers.PointCloud
                /apollo/sensor/camera/front_6mm/image                  469 messages: apollo.drivers.Image
                /apollo/sensor/camera/front_12mm/image                 471 messages: apollo.drivers.Image
                /apollo/sensor/gnss/ins_stat                           118 messages: apollo.drivers.gnss.InsStat
                /apollo/sensor/gnss/best_pose                           59 messages: apollo.drivers.gnss.GnssBestPose
                /apollo/sensor/gnss/rtk_eph                             49 messages: apollo.drivers.gnss.GnssEphemeris
```

**Comment by vilon888** [comment]
hi @daohu527 ， could you provide the link to to download perception_test_v1 Vehicle configuration? Thanks!
